src/components/switch/index.tsx

```diff
@@ -121,8 +120,8 @@ class Switch extends BaseComponent {
   }
 
   getSwitchStyle() {
-    const {value, onColor, offColor, style: propsStyle, disabled, disabledColor} = this.getThemeProps();
-    const style = [this.styles.switch];
+    const {value, onColor, offColor, style: propsStyle, disabled, disabledColor} = this.props;
+    const style: object[] = [this.styles.switch];
```

Comment:
Try to see how we define `Style` elsewhere in the props of other TS components.
BTW, it is more important that the actual `style` prop is fixed, but the same can be done here.

Reply:
Very nice!